.github/workflows/test-examples.yml

```diff
@@ -1,10 +1,13 @@
 name: Test Examples
 
-on:
-  push:
-    branches:
-      - master
+# on:
+#   push:
+#     branches:
+#       - master
```

Comment:
Should these be left uncommented, or why can't we just remove them?

Reply:
Ouch that is a commit that leaked in from another branch. My bad, I used git wrongly. Good catch

Reply:
fixed this one.